src/subscription_manager/cp_provider.py

```diff
@@ -182,56 +182,56 @@ def get_keycloak_auth_cp(self, token):
         self.set_token(access_token)
 
         self.keycloak_auth_cp = connection.UEPConnection(
+            host=self.server_hostname,
+            ssl_port=self.server_port,
+            handler=self.server_prefix,
+            proxy_hostname=self.proxy_hostname,
+            proxy_port=self.proxy_port,
+            proxy_user=self.proxy_user,
+            proxy_password=self.proxy_password,
+            username=None,
+            password=None,
+            correlation_id=self.correlation_id,
+            no_proxy=self.no_proxy,
+            restlib_class=self.restlib_class,
+            token=self.token,
+            client_version=self.get_client_version()
+        )
+        return self.keycloak_auth_cp
+
+    def get_basic_auth_cp(self):
+        if not self.basic_auth_cp:
+            self.basic_auth_cp = connection.UEPConnection(
                 host=self.server_hostname,
                 ssl_port=self.server_port,
                 handler=self.server_prefix,
                 proxy_hostname=self.proxy_hostname,
                 proxy_port=self.proxy_port,
                 proxy_user=self.proxy_user,
                 proxy_password=self.proxy_password,
-                username=None,
-                password=None,
+                username=self.username,
+                password=self.password,
```

Comment:
This looks like functional change.

Reply:
No, it's fine; it's the github diff viewer that is somehow confused: the "old" lines belong to the ` self.keycloak_auth_cp = connection.UEPConnection(` block, which is shown as being added as new few lines above.

If you check the content of the single commit you can notice there is no content change (and also `git diff -w` shows that too).